doc/man1/enc.pod

```diff
@@ -54,47 +54,47 @@ List all supported ciphers.
 
 =item B<-in filename>
 
-the input filename, standard input by default.
+The input filename, standard input by default.
```

Comment:
Side comment: hmmm, there are other pods where we start these descriptions with lower case first letter...  it was probably a design choice, but hasn't been followed in every (just a lot) of other apps manuals, and doesn't agree with how man pages are usually written.  So, good catch, and we all need to make a mental note to fix the other pages at some point.